### DISCUSSION TITLE
"Ask AI" page for mkdocs-material

### DISCUSSION THREAD

**Discussion by anorprogrammer** [squidfunk/mkdocs-material#6384] [Ideas]
Nowadays, most docs integrate with ai and they add a "ask ai" page (similar to ChatGPT). Can you add a similar page for mkdocs-material as well?

Docs page

![image](https://github.com/squidfunk/mkdocs-material/assets/77531974/f6ce02fa-844c-4243-8b78-7685c12fd18e)

Ask AI page

![image](https://github.com/squidfunk/mkdocs-material/assets/77531974/b6d4176c-e05d-41be-939c-833f32fc1c35)

**Comment by squidfunk** [comment]
Thanks for asking! We're currently [reworking the search](https://github.com/squidfunk/mkdocs-material/issues/6307) implementation, and will provide a simple way to connect the search interface to a third-party AI bot of your choosing. Since this involves hosting and a server, some effort for setting this up is necessary.
